# -*- coding: utf-8 -*-
"""Untitled0.ipynb

Automatically generated by Colaboratory.

Original file is located at
    https://colab.research.google.com/drive/1TGi38QXUpGhK2b6uBPpS43Ih3HgqcdFz
"""
import mysql.connector
from mysql.connector import Error
from google.cloud.sql.connector import connector

def create_server_connection():
    connection = None
    try:
        conn = connector.connect(
                "visitrack-314902:us-central1:visitrack-user-db", 
                "pymysql",
                user="root",
                password="12345678",
                db="visitrack-user-db"
        )
        print("MySQL Database connection successful")
    except Error as err:
        print(f"Error: '{err}'")

    return conn

create_server_connection()